clap_mangen/src/render.rs

```diff
@@ -99,9 +99,11 @@ pub(crate) fn options(roff: &mut Roff, cmd: &clap::Command) {
             (None, None) => vec![],
         };
 
-        if let Some(value) = &opt.get_value_names() {
-            header.push(roman("="));
-            header.push(italic(&value.join(" ")));
+        if opt.get_action().takes_values() {
```

Comment:
This seems suspicious. #4432 showed working man pages. We should know what the difference is between the old issue and new issue and discuss it in this PRs issue for which route to go.

Reply:
Interesting. Yeah i'll look at that.